### PR TITLE
fix: Ember is not defined error

### PR DESCRIPTION
**Pull request by wozny1989**
# Tasks:
 - [x] Fix error `Ember is not defined` - import Ember during library version register

### CODE REVIEW

**Review by Rxbsxn — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -464,7 +464,7 @@ module.exports = {
 
     let versionTree = writeFile(
       'ember-paper/register-version.js',
-      `Ember.libraries.register('Ember Paper', '${version}');`
+      `const Ember = require('ember').default;Ember.libraries.register('Ember Paper', '${version}');`
```
Comment (mansona) [inline index.js:467]: I think I would prefer just to remove this functionality 🤔 I don't think we need to define the Ember-Paper version in the client side for any good reason

File: package.json [hunk 2]
```diff
@@ -13,13 +13,14 @@
     "start": "ember serve",
     "test": "npm-run-all lint test:*",
     "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
+    "test:ember-compatibility": "ember try:each",
+    "postinstall": "patch-package"
```
Comment (mansona) [inline package.json:17]: we really can't be adding a patch-package post install to this 🙈

File: package.json [hunk 3]
```diff
@@ -42,14 +43,15 @@
...
+    "patch-package": "^6.5.1",
     "polyfill-nodelist-foreach": "^1.1.2",
     "propagating-hammerjs": "^2.0.1",
     "resolve": "^1.13.1",
     "sass": "^1.23.7",
     "tinycolor2": "^1.4.1"
   },
   "devDependencies": {
-    "@ember/jquery": "^1.1.0",
+    "@ember/jquery": "2.0.0",
```
Comment (mansona) [inline package.json:54]: we should probably be removing `@ember/jquery`. Is there anything useful that it's actually being used for?

**Review by mansona — CHANGES_REQUESTED** [review]
So I don't think we should probably do these changes 🙈 I understand that you're trying to fix something here but I would rather a) just plainly remove the functionality b) fix it upstream or c) upgrade to more standard ways of doing it.

I hope all that makes sense?

**Comment by wozny1989** [comment]
@mansona You're right, I just removed functionality with register version if is no super required 👍

**Review by mansona — APPROVED** [review]
looks great 🎉

**Pull request merged**